apps/ui/src/networks/common/graphqlApi/index.ts

```diff
@@ -300,19 +300,22 @@ export function createApi(uri: string, networkId: NetworkID, opts: ApiOptions =
       spaceIds: string[],
       { limit, skip = 0 }: PaginationOpts,
       current: number,
-      filter: 'any' | 'active' | 'pending' | 'closed' = 'any',
+      filters: ProposalsFilter,
       searchQuery = ''
     ): Promise<Proposal[]> => {
-      const filters: Record<string, any> = {};
-      if (filter === 'active') {
+      const state = filters?.state || 'any';
+
+      if (state === 'active') {
         filters.start_lte = current;
         filters.max_end_gte = current;
-      } else if (filter === 'pending') {
+      } else if (state === 'pending') {
         filters.start_gt = current;
-      } else if (filter === 'closed') {
+      } else if (state === 'closed') {
         filters.max_end_lt = current;
       }
 
+      delete filters?.space;
```

Comment:
Why do we delete space there? Shouldn't we delete `state`? And we should probably delete it in `offchain` as well.